HCCGo/app/js/GenericClusterInterface.js

```diff
@@ -12,25 +12,102 @@ var GenericClusterInterface = function(connectionService, $q) {
 }
 
 GenericClusterInterface.prototype.getJobs = function() {
-  
+
   // return a promise if the jobs info are found
-  
+
+}
+
+function CSVToArray( strData, strDelimiter ){
```

Comment:
I think when I was initially looking I just found the first thing that popped up, I'll see if I can find one and use that instead.